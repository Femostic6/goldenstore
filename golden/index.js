
const toggleBtn = document.querySelector(".toggle_btn")
const toggleBtnIcon = document.querySelector(".toggle_btn i")
const dropDownMenu = document.querySelector(".dropdown")
const order = document.getElementById("buttt")
const formHeading = document.querySelector("form-heading")
const contactForm = document.querySelectorAll("contact-forn-input")
toggleBtn.onclick = function(){
    dropDownMenu.classList.toggle("open")
    const isOpen = dropDownMenu.classList.contains("open")
toggleBtnIcon.classList = isOpen
 ?"fa-solid fa-xmark"
 : "fa-solid fa-bars"
}



    
function sendMessage(){     
     var message = "Your Delivery Details";
     alert( "ITEM ADDED TO CART");  
    setTimeout(function() {
        window.location.href = "./contactus.html?message=" + message + "&fromButton=true";
    }, 100);
    
   
}
window.onload = function() {

    var params = new URLSearchParams(window.location.search);
    var message = params.get('message');
    var fromButton = params.get('fromButton');

    if (message && fromButton === 'true') {
        document.getElementById('message').innerHTML = message;
    }
    else{
        document.getElementById('message').innerHTML = "Contact Us"
    }
}


document.getElementById("add").addEventListener("dblclick", function(db){
   db = alert("Your Item Has Been Added To Cart")
    
},)


