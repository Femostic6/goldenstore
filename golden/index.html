<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.css" 
    integrity="sha512-DTOQO9RWCH3ppGqcWaEA1BIZOC6xxalwEsw9c2QQeAIftl+Vegovlnee1c9QX4TctnWMn13TZye+giMm8e2LwA==" 
    crossorigin="anonymous" referrerpolicy="no-referrer" />
    <link rel="stylesheet" href="style.css">
    <meta name="Author" content="Ayodele Oluwafemi">
    <title>Golden store</title>

</head>
<body >
    <header>
        <div class="navbar">
            <div class="logo">
                <a href="./index.html"> <h1>Golden Store</h1></a>
            </div>
            <ul class="links">
                <li><a href="#">Home</a></li>
                <li><a href="./about.html">About</a></li>
                <li><a href="./contactus.html">Contact Us</a></li>
            
            </ul>
            <div class="toggle_btn">
                <i class="fa-solid fa-bars"></i>
            </div>
        </div>
        <div class="dropdown">
            <li><a href="#">Home</a></li>
            <li><a href="./about.html">About</a></li>
            <li><a href="./contactus.html">Contact Us</a></li>
        </div>
    </header>
    <main>
        <div id="mainbody">
            <img class="image" src="./images/iphone.png" alt="iphone">
            <div>
            <p class="details">
                Apple IPhone 15 Pro Max 512gb - Nano Sim - Natural Titanium
            </p>
            <h3 class="price">₦ 2,000,000</h3>
            <h3 ><span class="previous_price">₦ 3,300,000 </span> <span class="reduction">-17%</span></h3>
            <button onclick="sendMessage()" class="button" id="buttt" >ADD TO CART</button>

        </div>
             

        </div>
    </main>
    
    <script src="index.js"></script>
</body>
</html>